daos/mysql/Items.php

```diff
@@ -329,11 +329,11 @@ public function hasThumbnail($thumbnail) {
      * @param string $icon file
      */
     public function hasIcon($icon) {
-        $res = \F3::get('db')->exec('SELECT count(*) AS amount
-                   FROM '.\F3::get('db_prefix').'items 
-                   WHERE icon=:icon',
+        $res = \F3::get('db')->exec('SELECT id
```

Comment:
You even shouldn't return the ID, just do a `SELECT 1 FROM items WHERE icon=:icon LIMIT 1` :wink: